docker_setup.sh

```diff
@@ -115,16 +115,7 @@ docker run --name pause -td --restart unless-stopped \
 	-p $gui_port:$gui_port \
```

Comment:
Maybe add the following line over here? I have to update the script every time I test.
```
--hostname `hostname` \
```

Reply:
or can we use the cpiface options to pass this if its not important?

Reply:
actually central branch had this and its merged